specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Dataset.json

```diff
@@ -688,6 +832,48 @@
         "location"
       ]
     },
+    "XmlDataset": {
+      "x-ms-discriminator-value": "Xml",
+      "description": "Xml dataset.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Dataset"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Xml dataset properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/XmlDatasetTypeProperties"
+        }
+      }
+    },
+    "XmlDatasetTypeProperties": {
+      "description": "Xml dataset properties.",
+      "type": "object",
+      "properties": {
+        "location": {
+          "$ref": "#/definitions/DatasetLocation",
+          "description": "The location of the json data storage."
+        },
+        "encodingName": {
+          "type": "object",
```

Comment:
Is this really an object? It seems like encodingName should be a string

Reply:
As far as I know, some properties on ADF side can accept both strings and objects. I am not sure this is such case or not. Seems not. @drosales57 , can we change it to string type?